docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,62 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algortihms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the `scikit-learn transformers
+<https://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn.base.TransformerMixin>`_
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it.
+A :code:`fit_transform` might not always be available, in which case the separate
```

Comment:
I think that `fit_transform` should be provided automatically by `TransformerMixin`, please double check whether `CorrelationRemover` supports it (I think it might?).

Reply:
You are correct! `CorrelationRemoer` supports `fit_transform`, I overlooked this, thanks :)